services/market/package.json

```diff
@@ -28,6 +28,7 @@
     "eslint-fix": "eslint --fix ./api ./knowledge ./services ./test ./*.js",
     "clean": "rm -rf ./node_modules",
     "watch": "supervisor -w . -i ./node_modules app.js",
+    "test:unit": "./node_modules/.bin/jest --config=jest.config.unit.js --detectOpenHandles --forceExit",
```

Comment:
Are those really needed here? ` --detectOpenHandles --forceExit`

Reply:
They are necessary to exit the test successfully.